src/styles/main.scss

```diff
@@ -1,3 +1,11 @@
+/*********
+Global style guide
+_ If a block is nested or the HTML is generated, the style will not be applicable to the component style and will need to be written here
+_ There are some Helpers class among this file. Search for "HELPER CLASS" to find them.
+_ The style attribute does not work on the HTML block, as the HTML is sanitize. If is needed to style a specific HTML tag, is possible to add a class and add the style here.
+*********/
```

Comment:
A few grammar/spelling bits here. Here's a corrected version:

Global style guide
_ If a block is nested or the HTML is generated, the style will not be applicable to the component style and will need to be written here.
_ There are some helper classes below. Search for "HELPER CLASS" to find them.
_ The style attribute does not work on the HTML block, as the HTML is sanitised. If you need to style a specific HTML tag, you can add a class for the style here.